.github/workflows/build-opensearch-image.yaml

```
@@ -0,0 +1,33 @@
+name: Opensearch Docker Image
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - server/build/Dockerfile.opensearch
+      - .github/workflows/build-opensearch-image.yml
```

nit: either this or change the filename to `yml` (for consistency).
```suggestion
      - .github/workflows/build-opensearch-image.yaml
```

---

Good catch, thank you!